content/docs/reference/cookies.mdx

```diff
@@ -193,13 +197,7 @@ COOKIE_SECURE=false
 </TabItem>
 <TabItem value="Enterprise" label="Enterprise">
 
-Configure **Cookie Secure** with the **HTTPS only** toggle button in the Console. The button has three states:
-
-- **Unset** ("-") uses the value in your configuration file
-- **Checkmark** sets `cookie_secure` to `true`
-- **Empty** sets `cookie_secure` to `false`
-
-![Configure Cookie Secure in the Console](./img/cookies/cookie-settings.png)
+As of v0.25, we've removed support for the **Cookie Secure** setting in the Enterprise Console (in the Console, this setting was called **Javascript Security**). Now, this setting is always enabled by default.
```

Comment:
As far as I can tell, the part "Now, this setting is always enabled by default" is not correct. I think the UI was removed but not the backend logic, so Console may continue to control this setting (using whatever value is still in the database).

```suggestion
As of v0.25, we've removed the **Cookie Secure** setting from the Enterprise Console (in the Console, this setting was called **HTTPS only**).
```